classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
```

Comment:
- [ ] Great work. Kindly add students as `attr_reader`

Reply:
@tufoinnkuo10 you didn't specify the reason why i should add this, could you please provide more context. Thank you